source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. An example would be a lightbulb, a Chromecast, or a smart hub.
+
+Home Assistant has a **discovery** feature, which scans for new devices as they’re added and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices like cell phones that can move around.
+
+## Entities
+
+Because many devices do more than one thing, most things in Home Assistant use  **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
```

Comment:
I think it is more clear to somewhere put up that devices can provide other devices or entities.

Reply:
If I added "A device can also provide other devices", would that be true and fixing the issue?

Reply:
Yeah or: 

`A device can provide, one or more, entities or other devices it is connected to.`